scripts/st2bootstrap-el8.sh

```diff
@@ -692,7 +692,7 @@ EOT"
 
 install_st2chatops() {
   # Temporary hack until proper upstream fix https://bugs.centos.org/view.php?id=13669
```

Comment:
BTW @punkrokk when adding EL8 scripts, did you verify if this is still an issue?

Looks like this block was just copy-pasted from the EL7 scripts.
https://bugs.centos.org/view.php?id=13669

Reply:
At least CentOS8 has `http-parser` in the default AppStream repository.

Reply:
What do you mean by `verify if this is still an issue`? I was able to install it when I shipped this.